netlify.toml

```diff
@@ -63,6 +63,11 @@
   to = "/helium-tokens/hnt"
   status = 301
   force = true
+[[redirects]]
+  from = "/solana/migration/ledger"
+  to = "/solana/migration/wallet-user"
+  status = 301
+  force = true
```

Comment:
Can we put the `#` section headers here? 

e.g., `/wallets#for-ledger-wallet-users` to `/solana/migration/wallet-user#for-ledger-wallet-users`?